### PR TITLE
[Merged by Bors] - fix: rename Irreducible.nat_degree_le_two

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by adomani — APPROVED** [review]
Good catch!  Could you add the deprecation date, please?

File: Mathlib/Analysis/Complex/Polynomial.lean [hunk 1]
```diff
@@ -219,5 +219,7 @@ lemma Irreducible.degree_le_two {p : ℝ[X]} (hp : Irreducible p) : degree p ≤
     rwa [isUnit_iff_degree_eq_zero.1 hq, add_zero]
 
 /-- An irreducible real polynomial has natural degree at most two. -/
-lemma Irreducible.nat_degree_le_two {p : ℝ[X]} (hp : Irreducible p) : natDegree p ≤ 2 :=
+lemma Irreducible.natDegree_le_two {p : ℝ[X]} (hp : Irreducible p) : natDegree p ≤ 2 :=
   natDegree_le_iff_degree_le.2 hp.degree_le_two
+
+@[deprecated] alias Irreducible.nat_degree_le_two := Irreducible.natDegree_le_two
```
Comment (Ruben-VandeVelde) [inline Mathlib/Analysis/Complex/Polynomial.lean:225]:
```suggestion
@[deprecated] -- 2024-02-18
alias Irreducible.nat_degree_le_two := Irreducible.natDegree_le_two
```

**Comment by adomani** [comment]
Thanks!

maintainer merge

**Comment by adomani** [comment]
maintainer merge

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**